libnavui-dropin/src/main/res/layout/mapbox_info_panel_header_arrival_layout.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:background="@drawable/mapbox_bg_info_panel"
+    tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout"
+    tools:layout_height="@dimen/mapbox_infoPanel_peekHeight">
+
+    <androidx.appcompat.widget.AppCompatTextView
+        style="@style/DropInInfoPanelHeadlineTextAppearance"
```

Comment:
Let us remove this style from XML and apply it to the text at runtime. Refs: https://github.com/mapbox/mapbox-navigation-android/pull/6273/files#r969215976